processor/eventfilter/eventfilter.go

```diff
@@ -24,20 +24,22 @@ var pkgLogger = logger.NewLogger().Child("eventfilter")
 func GetSupportedMessageTypes(destination *backendconfig.DestinationT) ([]string, bool) {
 	var supportedMessageTypes []string
 	if supportedTypes, ok := destination.DestinationDefinition.Config["supportedMessageTypes"]; ok {
-		if supportedTypeInterface, ok := supportedTypes.([]interface{}); ok {
-			supportedTypesArr := misc.ConvertInterfaceToStringArray(supportedTypeInterface)
-			for _, supportedType := range supportedTypesArr {
-				var skip bool
-				switch supportedType {
-				case "identify":
-					skip = identifyDisabled(destination)
-				default:
-				}
-				if !skip {
-					supportedMessageTypes = append(supportedMessageTypes, supportedType)
+		if supportedTypesMap, ok := supportedTypes.(map[string]interface{}); ok {
+			if supportedTypeInterface, ok := supportedTypesMap["cloud"].([]interface{}); ok {
+				supportedTypesArr := misc.ConvertInterfaceToStringArray(supportedTypeInterface)
+				for _, supportedType := range supportedTypesArr {
+					var skip bool
+					switch supportedType {
+					case "identify":
+						skip = identifyDisabled(destination)
```

Comment:
when we have supportMessageTypes when again why need again identifyDisabled (enableServerSideIdentify)?